src/backend/tests/unmocked/organization.test.ts

```diff
@@ -16,26 +23,64 @@ describe('Team Type Tests', () => {
     await resetUsers();
   });
 
+  describe('Set Images', () => {
+    it('Fails if user is not an admin', async () => {
+      await expect(OrganizationsService.setImages([], await createTestUser(wonderwomanGuest, orgId), orgId)).rejects.toThrow(
+        new AccessDeniedAdminOnlyException('update images')
+      );
+    });
+
+    it('Fails if an organization does not exist', async () => {
+      await expect(OrganizationsService.setImages([], await createTestUser(batmanAppAdmin, orgId), '1')).rejects.toThrow(
+        new HttpException(400, `Organization with id: 1 not found!`)
+      );
+    });
+
+    it('Succeeds and updates all the images', async () => {
+      const testBatman = await createTestUser(batmanAppAdmin, orgId);
+      const testFiles = [
+        { originalname: 'image1.png', buffer: Buffer.from('') },
+        { originalname: 'image2.png', buffer: Buffer.from('') }
+      ] as Express.Multer.File[];
+
+      (uploadFile as Mock).mockImplementation((file) => {
+        return Promise.resolve({ id: `uploaded-${file.originalname}` });
+      });
+
+      await OrganizationsService.setImages(testFiles, testBatman, orgId);
+
+      const organization = await prisma.organization.findUnique({
+        where: {
+          organizationId: orgId
+        }
+      });
+
+      expect(organization).not.toBeNull();
+      expect(organization!.interestedinApplyingImage).toMatch(/^uploaded-image1\.png$/);
+      expect(organization!.exploreAsGuestImage).toMatch(/^uploaded-image2\.png$/);
```

Comment:
do you need the exclimation marks after organization?

Reply:
Can you also add another test of updating them. Maybe even in the same test but upload a new picture for just the second image and see if it updates as expected